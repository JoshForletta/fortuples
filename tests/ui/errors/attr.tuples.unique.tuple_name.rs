use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    #[tuples::tuple_name(A)]
    #[tuples::tuple_name(B)]
    impl Container<#Tuple> {}
}

fn main() {}
