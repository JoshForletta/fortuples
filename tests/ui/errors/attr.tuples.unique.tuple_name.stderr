error: `tuple_name` should be set only once
 --> tests/ui/errors/attr.tuples.unique.tuple_name.rs:7:15
  |
7 |     #[tuples::tuple_name(B)]
  |               ^^^^^^^^^^

error: the first definition was here
 --> tests/ui/errors/attr.tuples.unique.tuple_name.rs:6:15
  |
6 |     #[tuples::tuple_name(A)]
  |               ^^^^^^^^^^
